tests/test_utils/alloc.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017-2018 Data61, CSIRO
```

Comment:
If this is a new file, then copyright should be 2019.

Reply:
```suggestion
# Copyright 2017-2019 Data61, CSIRO
```

Reply:
The paradigm we've been using is that any file created in 2019 is copyrighted just for 2019, not from the start of the project. Any modifications in 2020 would then change the copyright to 2019-2020.

Reply:
👍 

```suggestion
# Copyright 2019 Data61, CSIRO
```

FYI, the "suggestions" button on GitHub (the little page icon contain a +/-) makes it easy to be clear exactly what's meant for single-line changes like this and easy for the PR author to apply.

Reply:
Thanks, I got Terry to show me where that *tiny* little button lives!!